modules/Milestone/MilestoneController.js

```diff
@@ -191,6 +191,12 @@ module.exports = {
       milestone.proposalNearTransactionHash = proposalNearTransactionHash;
       milestone.isNearProposalValid = true;
 
+      if (grantConfig.skipMilestoneInterviewAndApproval) {
+        milestone.dateInterviewScheduled = new Date();
+        milestone.dateInterview = new Date();
```

Comment:
@sandoche I'll approve it but this feels like a hack (and maybe should have comments to document what is going on) and I wonder whether this will make any data confusing. If people go back and look at the data, they won't know that these interviews didn't happen, right? Unless they happened to know (and be thinking about) the fact that the dates are identical and that there was this weird hack.  Unless I'm misunderstanding.

Reply:
You are correct, it's a bit hacky indeed. I think to have the full modularity of the code we would need to rethink the architecture but since it's not a priority I went the fastest way. I'll add a comment before merging.